Makefile

```diff
@@ -18,14 +17,15 @@ build-frontend:
 
 start-database:
 	docker-compose up -d
+	pipenv run python3 main.py create-database
```

Comment:
Funktioniert nicht. Man müsste erstmal in den backend Ordner wechseln, und `pipenv sync` ausführen. Deswegen hielt ich es für praktischer, es beim start von dem server auszuführen.